shared/store/wallet.ts

```diff
@@ -128,7 +130,8 @@ export const fetchTokens = createAsyncThunk(
   'wallet/fetchTokens',
   async ({ size = 200, address, client }: { size?: number, address: string, client: WhaleApiClient }): Promise<{ tokens: AddressToken[], allTokens: TokenData[], utxoBalance: string }> => {
     const tokens = await client.address.listToken(address, size)
-    const allTokens = await client.tokens.list(size)
```

Comment:
Does it handle also possible swap tokens?

Reply:
yes i believe. I checked the swap tokens and dAmz/v1 is being filtered out